packages/ui-kit/react_import.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react'
```

Comment:
What is this for?

Reply:
https://github.com/egoist/tsup/issues/390#issuecomment-933488738
I'll add comment to the code for future reviewers